src/devices/Ht1632/README.md

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
This file should not be empty, but include a small documentation on how to use the chip (idealy with a sample wiring diagram), references to the relevant data sheets, etc.